.github/workflows/testBuildHybrid.yml

```diff
@@ -224,8 +224,8 @@ jobs:
           java-version: '17'
 
       - name: Setup Ruby
-        # v1.190.0
-        uses: ruby/setup-ruby@a6e6f86333f0a2523ece813039b8b4be04560854
```

Comment:
There's a few more updates in this file

Reply:
from https://github.com/Expensify/App/pull/59412/files#diff-db05e56613d35c8f4e00f2803e59d327df466973d5c7c3176db192205519027b